packages/arch3-core/src/signingarchwayclient.spec.ts

```diff
@@ -378,5 +378,29 @@ describe('SigningArchwayClient', () => {
         client.disconnect();
       });
     });
+
+    it('gracefully handles withdrawing when no rewards found', async () => {
+      const [wallet, accounts] = await getWalletWithAccounts();
+      const client = await SigningArchwayClient.connectWithSigner(archwayd.endpoint, wallet, clientOptions);
+
+      const rewardsAddress = accounts[3].address;
```

Comment:
Here, you can use `accounts[4]` as this account shouldn't have any rewards assigned to it during any other tests.